### PR TITLE
[CWS][SEC-5001] Add container.created_at attribute

### PR DESCRIPTION
**Pull request by lebauce**
### What does this PR do?

Allow writing SECL rules against container creation time through the new `container.created_at`
field, similar to the existing `process.container_at` field.

Example:  `container.created_at > 3s && exec.file.path == "/path/to/a/file"`

The container creation time is also reported in the sent events.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by alai97 — APPROVED** [review]
Looks good for docs!

File: pkg/security/probe/field_handlers.go [hunk 1]
```diff
@@ -156,6 +156,18 @@ func (fh *FieldHandlers) ResolveContainerID(ev *model.Event, e *model.ContainerC
 	return e.ID
 }
 
+// ResolveContainerCreatedAt resolves the container creation time of the event
+func (fh *FieldHandlers) ResolveContainerCreatedAt(ev *model.Event, e *model.Process) int {
+	if e.ContainerCreatedAt == 0 {
+		if entry, _ := fh.ResolveProcessCacheEntry(ev); entry != nil {
+			if cgroup, _ := fh.resolvers.CgroupsResolver.Get(entry.ContainerID); entry != nil {
```
Comment (safchain) [inline pkg/security/probe/field_handlers.go:163]: should we check that the entry has a ContainerID before trying to request the CgroupResolver ?

Comment (safchain) [reply pkg/security/probe/field_handlers.go:163]: wondering if the `e` process context is not already the process cache entry ? if so you could remove the `ResolveProcessCacheEntry` call

**Comment by safchain** [comment]
could you add a functional test ?

File: pkg/security/resolvers/cgroup/model/model.go [hunk 2]
```diff
@@ -98,3 +99,8 @@ func (cgce *CacheEntry) SetTags(tags []string) {
 func (cgce *CacheEntry) NeedsTagsResolution() bool {
 	return len(cgce.ID) != 0 && !cgce.WorkloadSelector.IsEmpty()
 }
+
+// GetCreationTime returns the root pid of a cgroup
+func (cgce *CacheEntry) GetCreationTime() uint64 {
```
Comment (paulcacheux) [inline pkg/security/resolvers/cgroup/model/model.go:104]: Nit: why a Get function when the underlying property is already public ?

**Review by paulcacheux — APPROVED** [review]

File: pkg/security/resolvers/cgroup/model/model.go (hunk 2 above)
Comment (lebauce) [reply pkg/security/resolvers/cgroup/model/model.go:104]: Because I was too lazy to remove the function when figuring out this is useless :-)
I removed it

**Pull request merged**